README.md

```diff
@@ -1 +1,31 @@
-# angular-what-is-angular-readme
+# What is Angular and why does it exist?
+
+## Overview
+
+AngularJS (commonly referred to as just "Angular") is an open-source framework released in 2009, being mainly maintained by Google as well as a massive community of developers.
+
+It aims to solve the problems faced when creating single page applications, implementing a MVVM pattern to separate presentation and business logic.
+
+## Objectives
+
+- Describe what Angular's role in the web platform is
+- Describe the issues Angular looks to solve
+- Describe how Angular thinks in terms of Web Components to create encapsulation and reusable components
+
+## Traditional Rendering
+
+Before mainstream JavaScript usage, everything on the client was determined by the server. Routing and views were defined in backend code, meaning view logic was frequently mixed in with business logic, providing the end-user with a poor user experience.
```

Comment:
This is too technical - explain the problem here in more general terms. For example, "traditional back end frameworks like Ruby on Rails process requests and send back responses to the client. For example, if you go to www.nytimes.com, the server loads a list of articles and sends back the response. The problem with that is..." 

In general, this paragraph just lacks clarity. Ideally, my mom should be able to understand it - she doesn't know much about computers, but is a smart person. Again, the "Why Use  A Web Framework" intro would be accessible for her. Avoid terms like "view logic" and "business logic" - come up with examples or analogies instead.